.github/workflows/build-apk.yml

```diff
@@ -0,0 +1,69 @@
+name: Build APK
+
+on:
+  push:
+    branches:
+      - main
+      - feature/set-up-github-APK-button
+  pull_request:
```

Comment:
Same as for push. From M1 instructions:

> _You must set up your repo with a GitHub workflow to build the app's APK such that, at the end of every Sprint, you can create a new release at the push of a button._ 

workflow_dispatch should be enough